src/main/java/com/vamae/repository/NewsRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.vamae.repository;
+
+import com.vamae.entity.News;
+import lombok.NonNull;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface NewsRepository extends MongoRepository<News, String> {
```

Comment:
`MongoRepository` already has methods `findById` and `findAll`, so remove the definitions